digiwf-libs/digiwf-camunda-prometheus/digiwf-camunda-prometheus-core/src/main/java/de/muenchen/oss/digiwf/camunda/prometheus/FniAndEdeMetricsProvider.java

```diff
@@ -23,16 +20,8 @@ public class FniAndEdeMetricsProvider implements MetricsProvider {
     @Override
     public void updateMetrics() {
         // Get total number of FNIs
-        this.fniCount
-            .labels("total", "total")
-            .set(this.managementService.createMetricsQuery().name("activity-instance-start").sum());
-
-        // Get FNIs by process definition (this only includes the currently deployed definitions and may be lower than the total count)
-        repositoryService.createProcessDefinitionQuery().list().forEach(processDefinition ->
-            this.fniCount
-                .labels(processDefinition.getKey(), processDefinition.getId())
-                .set(historyService.createHistoricActivityInstanceQuery().processDefinitionId(processDefinition.getId()).count())
-        );
+        this.fniCount.set(this.managementService.createMetricsQuery().name("activity-instance-start").sum());
```

Comment:
Sollten wir hier nicht eher `root-process-instance-start` nehmen? Warum die FNI 🤔

Reply:
Kein blocker der Kommentar, ich frag mich nur, was wir hier messen und warum

Reply:
Den `root-process-instance-start` fand ich schon interessant, sodass man sieht, wie viele Prozesse allgemein gestartet werden. Meinst du nicht?

Reply:
Reicht uns das nicht das Start-Event: https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-libs/digiwf-camunda-prometheus/digiwf-camunda-prometheus-core/src/main/java/de/muenchen/oss/digiwf/camunda/prometheus/ExecutionEventReporter.java#L31
Ist ja mit ProcessDefinitionKey

Reply:
Das reicht ja. Dann passt es